lib/tapioca/generators.rb

```diff
@@ -1,6 +1,9 @@
 # typed: true
 # frozen_string_literal: true
 
+require "thor" # TODO: Remove me when logging logic has been abstracted.
```

Comment:
`generator.rb` previously contained `require "thor"` which was removed in this PR. Moved it to this file so it will be included in subsequent generators until it has been moved out into a logger.